DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -56,14 +56,67 @@
 </div>
 <div class="nhsuk-grid-row">
   <div class="nhsuk-grid-column-full">
-    <a class="nhsuk-button">
+    <details class="nhsuk-details nhsuk-expander">
+      <summary class="nhsuk-details__summary">
+        <span class="nhsuk-details__summary-text">
+          My Details
+        </span>
+      </summary>
+      <div class="nhsuk-details__text">
+        <dl class="nhsuk-summary-list">
+          <div class="nhsuk-summary-list__row account-summarylist_row">
+            <dt class="nhsuk-summary-list__key">
+              First name:
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.FirstName
+            </dd>
+          </div>
+
+          <div class="nhsuk-summary-list__row account-summarylist_row">
+            <dt class="nhsuk-summary-list__key">
+              Last name:
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.Surname
+            </dd>
+          </div>
+
+          @if (Model.User?.Length > 0)

```

Comment:
I think we probably want to show blank here if the email address doesn't exist so the user knows what they have an option to set it up